TPP.Core/Commands/Definitions/BadgeCommands.cs

```diff
@@ -253,18 +267,51 @@ public async Task<CommandResult> Pokedex(CommandContext context)
                     ResponseTarget = ResponseTarget.WhisperIfLong
                 };
             }
+            else if (mode.Equals(PokedexModeKantoooooooo))
+            {
+                ImmutableSortedDictionary<PkmnSpecies, int> ownedPokemons = (await _badgeRepo.CountByUserPerSpecies(user.Id));
+                int userOwnedRegionCount = ownedPokemons.Count(ownedPokemon =>
+                    ownedPokemon.Key.GetGeneration() == Generation.Gen1);
+                return new CommandResult
+                {
+                    Response = isSelf
+                        ? $"You have collected {userOwnedRegionCount}/151 distinct KANTOOOOOOOOOOOOOOO OhMyDog badge(s)"
+                        : $"{user.Name} has collected {userOwnedRegionCount}/151 distinct KANTOOOOOOOOOOOOOOO OhMyDog badge(s)"
+                };
+            }
+            else if (_pokedexModeRegions.ContainsKey(mode.ToLower()))
```

Comment:
just lowercasing the mode once at the beginning of the entire pokdex command would also benefit the other modes and avoid duplicating calls to `ToLower()`